supertokens_python/recipe/usermetadata/asyncio/__init__.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any, Dict
+
+from supertokens_python.recipe.usermetadata.recipe import UserMetadataRecipe
+
+
+async def get_user_metadata(user_id: str, user_context: Dict[str, Any]):
```

Comment:
```suggestion
async def get_user_metadata(user_id: str, user_context: Union[None, Dict[str, Any]] = None):
```

Reply:
`user_context` needs to be optional in all user exposed functions to keep the API clean. This is as opposed to recipe functions in which they are compulsory since those should never be called by the user directly anyway